org.uqbar.project.wollok/src/org/uqbar/project/wollok/interpreter/nativeobj/WollokJavaConversions.xtend

```diff
@@ -1,37 +1,42 @@
 package org.uqbar.project.wollok.interpreter.nativeobj
 
 import java.math.BigDecimal
+import java.time.LocalDate
 import java.util.Collection
 import java.util.List
 import java.util.Set
 import org.eclipse.xtext.xbase.lib.Functions.Function1
 import org.uqbar.project.wollok.interpreter.WollokInterpreter
 import org.uqbar.project.wollok.interpreter.WollokInterpreterEvaluator
 import org.uqbar.project.wollok.interpreter.core.WollokObject
+import org.uqbar.project.wollok.interpreter.core.WollokProgramExceptionWrapper
 
 import static org.uqbar.project.wollok.sdk.WollokDSK.*
-import org.uqbar.project.wollok.interpreter.core.WollokProgramExceptionWrapper
 
 /**
  * Holds common extensions for Wollok to Java and Java to Wollok conversions.
  * 
  * @author jfernandes
  */
 class WollokJavaConversions {
-	
-	def static asInteger(WollokObject it) { ((it as WollokObject).getNativeObject(INTEGER) as JavaWrapper<Integer>).wrapped }
-	
+
+	def static asInteger(WollokObject it) {
+		((it as WollokObject).getNativeObject(INTEGER) as JavaWrapper<Integer>).wrapped
+	}
+
 	def static isWBoolean(Object it) { it instanceof WollokObject && (it as WollokObject).hasNativeType(BOOLEAN) }
-	
-	def static isTrue(Object it) { it instanceof WollokObject && ((it as WollokObject).getNativeObject(BOOLEAN) as JavaWrapper<Boolean>).wrapped }
-	
+
+	def static isTrue(Object it) {
+		it instanceof WollokObject && ((it as WollokObject).getNativeObject(BOOLEAN) as JavaWrapper<Boolean>).wrapped
+	}
+
 	def static Object wollokToJava(Object o, Class<?> t) {
-		if (o == null) return null
-		if (t.isInstance(o)) return o
-		if (t == Object) return o
+		if(o == null) return null
+		if(t.isInstance(o)) return o
+		if(t == Object) return o
```

Comment:
I don't think so. I have Eclipse default formatting. There is a space between if and first parentheses, I use it all the time...